### ISSUE TITLE
problem with property html on AxisRendererCircular

### ISSUE THREAD

**Issue by symrad** [amcharts/amcharts5#1088]
I have a use case where I need custom labels on the AxisRendererCircular.
I use this simple configuration:

```ts
var xRenderer = am5radar.AxisRendererCircular.new(root, {
      minGridDistance: 10
});
xRenderer.labels.template.setAll({
      textType: "adjusted",
      html: "<div style='background-color:red'>{value1}-{category}</div>"
    });
```

The label is applied correctly, but i noticed that when the html property is used instead of text, the cpu goes from 0 to 15%.
Due to this problem I cannot customize the label with html.

I reproduce the problem using this demo:
https://www.amcharts.com/demos/variable-radius-radar/

https://jsfiddle.net/rqosugnj/3/ (with html property) CPU: 0
https://jsfiddle.net/rqosugnj/2/ (without html property) CPU: 0 to 20%

**Comment by zeroin** [comment]
Indeed, I see that ticking does not stop in case HTML is used. We will investigate this and will let you know.

**Comment by martynasma** [comment]
Fixed in [5.4.7](https://github.com/amcharts/amcharts5/releases/tag/5.4.7).

## [5.4.7] - 2023-09-18

### Added
- `panelResizer` property added to `StockPanel`. Holds an instance of an element used as a grip for resizing a panel.
- `originalEvent` added to all `XYCursor` events.
- `selectcancelled` event added to `XYCursor`.

### Fixed
- HTML axis label on a circular chart were causing unnecessary CPU usage in some setups.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
